.gitignore

```
@@ -13,3 +13,5 @@ Makefile
 compile_commands.json
 result
 result-*
+bin/
```

suggestion: also ignore generated lib files

Or maybe move these C++ ignores into the pkgdb tree?

```suggestion
lib/
```

---

I moved the ignore to the `pkgdb` tree. I was in a weird git state last night when I made these changes and it seems to be sorted out now.